tutorials/elixir-phoenix-on-kubernetes-google-container-engine/index.md

```diff
@@ -0,0 +1,444 @@
+    that once the cluster is running, you will be charged for the VM usage.
+
+3.  Configure gcloud to use your cluster by default, so you don't have to
+    specify it every time for the remaining gcloud commands.
+
+        gcloud config set container/cluster hello-cluster
+
+    Replace the name if you named your cluster differently.
+
+### Deploying to the cluster
+
+A production deployment comprises two parts: your Docker container, and a
+front-end load balancer (which also provides a public IP address.)
+
+We'll assume that you built the image to `gcr.io/${PROJECT_ID}/hello:v1` and
+you've created the Kubernetes cluster as described above.
+
+1.  Create a deployment
+
+        kubectl run hello-web --image=gcr.io/${PROJECT_ID}/hello:v1 --port 8080
+
+    This will run your image on a Kubernetes pod, which is the deployable unit
+    in Kubernetes. The pod opens port 8080, which is the port your Phoenix
+    application is listening on.
+
+    You may view the running pods using:
+
+        kubectl get pods
+
+2.  "Expose" the application by creating a load balancer pointing at your pod.
+
+        kubectl expose deployment hello-web --type=LoadBalancer --port 80 --target-port 8080
+
+    This creates a "service" resource pointing at your running pod. It listens
+    on the standard HTTP port 80, and proxies back to your pod on port 8080.
+
+3.  Obtain the IP address of the service by running:
+
+        kubectl get service
+
+    Initially, the "external IP" will be pending while Container Engine works
+    to procure an IP address for you. If you rerun the `kubectl get service`
+    command repeatedly, eventually the IP address will appear. You can then
+    point your browser at that URL to view the running application.
+
+Congratulations! Your application is now up and running!
+
+## Scale and update your application
+
+You'll now explore a few of the basic features of Kubernetes for managing your
+running app.
+
+### Setting the replica count
+
+Initially your deployment runs a single instance of your application. You may
+add more replicas using the `kubectl scale` command. For example, to add two
+additional replicas (for a total of three), run:
+
+    kubectl scale deployment hello-web --replicas=3
+
+Once the additional replicas are running, you can see the list of three pods
+by running:
+
+    kubectl get pods
+
+Kubernetes automatically allocates your running pods on the virtual machines
+in your cluster. You may configure pods in your deployment with specific
+resource requirements such as memory and CPU. See the
+[Kubernetes documentation](https://kubernetes.io/docs/home/) for more details.
+
+### Updating your application
+
+After you make a change to your app, redeploying is just a matter of
+building a new image and pointing your deployment to it.
+
+1.  Make a change to the app. (For example, modify the front page template.)
+
+2.  Perform a new build with a new version tag "v2":
```

Comment:
So this falls down pretty quick.  About the 4th time you build, you get tired of remembering how to modify the tag and v to use next.  Possibly something we could mention as an aside, but I tote this bash script with me in every k8s project I use.  
https://github.com/JustinBeckwith/cloudcats/blob/master/web/update.sh

Reply:
Yeah I get it, it's annoying to manage tags manually. But the alternative is, as you say, provide a script or something, which complicates the tutorial and makes the process more opaque and less informative.

Reply:
Fair point.  I'm ok with this as it is :)